handlers/handlers.go

```diff
@@ -79,16 +82,20 @@ type ServiceInfo struct {
 // ServiceSet is the base type for the HTTP handlers which combines multiple
 // mbtiles.DB tilesets.
 type ServiceSet struct {
-	tilesets map[string]*mbtiles.DB
-	Domain   string
-	Path     string
+	tilesets  map[string]*mbtiles.DB
+	templates *template.Template
+	Domain    string
+	Path      string
 }
 
 // New returns a new ServiceSet. Use AddDBOnPath to add a mbtiles file.
 func New() *ServiceSet {
-	return &ServiceSet{
-		tilesets: make(map[string]*mbtiles.DB),
+	s := &ServiceSet{
+		tilesets:  make(map[string]*mbtiles.DB),
+		templates: template.New("base"),
```

Comment:
In #48, you used `template.New("_base_")` - should this be consistent?

Reply:
Good catch!